bin/uglifyjs

```diff
@@ -135,7 +136,7 @@ if (program.output == "ast") {
 if (program.parse) {
     if (!program.parse.acorn && !program.parse.spidermonkey) {
         options.parse = program.parse;
-    } else if (program.sourceMap && program.sourceMap.content == "inline") {
+    } else if (content == "inline") {
```

Comment:
I'm little bit worried about this changes..
`ESLint: 'content' is not defined. (no-undef)`

Reply:
Oops!